Cargo.lock

```diff
@@ -417,9 +417,9 @@ dependencies = [
 
 [[package]]
 name = "openat-ext"
-version = "0.1.7"
+version = "0.1.8"
```

Comment:
For context, I usually split commits wrt. manifest changes vs lockfile updates vs code changes.
While that makes a single commit not anymore atomic / bisectable, from past experiences it helps cherry-picks and backports.